experiments/tf_trainer/common/tfrecord_input.py

```diff
@@ -56,21 +57,16 @@ def _input_fn_from_file(self, filepath: str) -> types.FeatureAndLabelTensors:
             {self._text_feature: [None],
             'sequence_length': []},
             {label: [] for label in self._labels})
-    # TODO: Remove long sentences.
+    # TODO(fprost_: Remove long sentences.
```

Comment:
Nit: Typo? (should it be '(fprost)')

Also, could you expand on what this means?